libs/partners/nvidia-ai-endpoints/langchain_nvidia_ai_endpoints/_common.py

```diff
@@ -485,67 +491,3 @@ def get_model_details(self, model: Optional[str] = None) -> dict:
         known_fns = self.client.available_functions
         fn_spec = [f for f in known_fns if f.get("id") == model_key][0]
         return fn_spec
-
-    def get_generation(
```

Comment:
Looks like these were all moved to chat model, and probably don't make sense for anything else inheriting from common?

Reply:
Yup! Just wanted to remove redundancy between the ChatNVIDIA and NVIDIAEmbeddings core components.